### PR TITLE
Hotfix for FORS2 image typing

### PR DESCRIPTION
**Pull request by profxj**
Branched off develop as I anticipated more changes than I found necessary.

### CODE REVIEW

**Review by rcooke-ast — APPROVED** [review]
Sorry - that's likely my fault!

**Comment by profxj** [comment]
(base) profx> py.test test_setups.py::test_setup_vlt_fors2 --disable-warnings 
============================= test session starts ==============================
platform linux -- Python 3.8.5, pytest-6.0.2, py-1.9.0, pluggy-0.13.1
rootdir: /data/Projects/Python/PypeIt
collected 1 item                                                               

test_setups.py .                                                         [100%]

======================== 1 passed, 4 warnings in 3.23s =========================

File: pypeit/scripts/chk_for_calibs.py [hunk 1]
```diff
@@ -137,15 +139,17 @@ def main(args):
...
+                                                           raise_error=False, cut_cfg=in_cfg)
         if not answers['pass'][i]:
             msgs.warn("Setup {} did not pass the calibration check!".format(setup))
 
     print('= RESULTS ============================================')
     # Print
     answers.pprint_all()
     print('======================================================')
+    # Remove setup_files
+    shutil.rmtree('setup_files')
```
Comment (kbwestfall) [inline pypeit/scripts/chk_for_calibs.py:152]: Do you want to always remove the setup files or would it make sense to make it a command-line option to keep them?

**Review by kbwestfall — APPROVED** [review]
Looks good!  Just one question about the change to `chk_for_calibs.py`.

File: pypeit/scripts/chk_for_calibs.py (hunk 1 above)
Comment (profxj) [reply pypeit/scripts/chk_for_calibs.py]: Good suggestion.  Added.

**Comment by profxj** [comment]
merging

**Pull request merged**